app/controllers/placements/schools/placements/build_controller.rb

```diff
@@ -0,0 +1,191 @@
+class Placements::Schools::Placements::BuildController < ApplicationController
+  before_action :setup_session, except: :new
+
+  def new
+    reset_session
+    @placement = build_placement
+    school.primary_or_secondary_only? ? handle_primary_or_secondary : handle_other
+  end
+
+  def add_phase
+    @placement = build_placement
+    @selected_phase = session[:add_a_placement]["phase"] || school.phase
+  end
+
+  def add_subject
+    setup_phase_navigation
+    build_or_retrieve_placement
+    assign_subjects_based_on_phase
+  end
+
+  def add_mentors
+    @placement = build_placement
+    @selected_mentors = find_mentors
+  end
+
+  def check_your_answers
+    session[:add_a_placement][:enable_phase_navigation] = true
+    @placement = initialize_placement
+    @phase = session[:add_a_placement]["phase"]
```

Comment:
```suggestion
    @phase = session.dig(:add_a_placement, "phase")
```

Reply:
You marked this as resolved, but I don't think it's changed.

Reply:
You're right, sorry about that!